models/repo_generate.go

```diff
@@ -39,6 +40,34 @@ func (gro GenerateRepoOptions) IsValid() bool {
 	return gro.GitContent || gro.Topics || gro.GitHooks || gro.Webhooks || gro.Avatar || gro.IssueLabels // or other items as they are added
 }
 
+func parseGiteaTemplate(tmpDir string) ([]glob.Glob, error) {
+	globs := make([]glob.Glob, 0)
+	gtPath := filepath.Join(tmpDir, ".giteatemplate")
+	if _, err := os.Stat(gtPath); err != nil {
+		return globs, err
+	}
+	content, err := ioutil.ReadFile(gtPath)
+	if err != nil {
+		return globs, err
+	}
+
+	lines := strings.Split(string(content), "\n")
```

Comment:
```suggestion
	lines := strings.Split(strings.ReplaceAll(string(content),"\r","\n"), "\n")
```

To support all kinds of EOL.

Reply:
Don't we have a super speedy way of converting line endings...

Reply:
Right! You did it yourself. 😁 
`util.NormalizeEOL()`

Reply:
Done